models/cflow.json

```diff
@@ -1,7 +1,8 @@
 {
     "fact_table": "cflow_facts",
     "label": "Cash Flow",
-    "description": "This is based on the CFB, CFR, CFA and CFAA return forms.",
+    "description": "Statement of Financial Position. The movement of cash into and out of the municipality.",
```

Comment:
missed this earlier - I don't think this is Statement of Financial Position. I think only bsheet is that.